widget/widgetAssets/js/iocExtractionConfigService.service.js

```diff
@@ -62,10 +64,40 @@
             "key",
             "jSONValue"
           ]
+        },
+        globalVariablesToKeyStoreMapping: {
+          "sfsp-excludelist-ips": ["Excludelist_IPs", "IP Address", true],
+          "sfsp-excludelist-urls": ["Excludelist_URLs", "URL", true],
+          "sfsp-excludelist-domains": ["Excludelist_Domains", "Domain", true],
+          "sfsp-excludelist-ports": ["Excludelist_Ports", "Port", true],
+          "sfsp-excludelist-files": ["Excludelist_Files", "File", true],
+          "sfsp-excludelist-cidr-ranges": ["NA", "CIDR Range", true],
+          "sfsp-excludelist-file-hashes":["NA", "File Hash", true],
+          "sfsp-indicator-type-mapping": ["Indicator_Type_Map", "NA", true]
         }
       }
     }
 
+
+    function getIndicatorRegex() {
+      return executeAction('cyops_utilities', 'get_regx_of_indicators', null, []);
+    }
+
+
+    function executeAction(connector_name, connector_action, userLoginId, payload) {
```

Comment:
executeAction --> executeConnectorAction

Reply:
The function name "executeAction" was chosen to avoid confusion with 'connectorService,' which calls `executeConnectorAction`. Instead of renaming "executeAction" to "executeConnectorAction," I suggest using "executeConnectorOperation."